### PR TITLE
Make 'types of advice' require at least one checked

### PR DESCRIPTION
**Pull request by neoeno**
See https://github.com/moneyadviceservice/mas-rad_core/pull/79

Also:

* Rename 'business split' to 'advice types'.
* Remove the 'other' type of advice.
* Remove 'supplementary advice' text that is no longer required after removing 'other'

### CODE REVIEW

**Comment by rnalexander** [comment]
:+1:

**Comment by munckymagik** [comment]
:+1:

File: app/views/self_service/firms/questionnaire/_business_split.html.erb [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 
-  <% t('questionnaire.retirement_advice.business_split.advice_options').each do |attribute, heading| %>
-    <div class="form__group-item">
-      <label>
-        <%= f.check_box attribute, class: "form__input t-#{attribute.to_s.dasherize}", aria_labelledby: "form-label-#{attribute} input-label-#{attribute}-b".dasherize %>
-        <%= raw heading %>
-      </label>
-    </div>
+  <%= f.form_row :advice_types do %>
+  <%= f.errors_for :advice_types %>
```
Comment (benbarnett) [inline app/views/self_service/firms/questionnaire/_business_split.html.erb]: Indentationiano

Comment (neoeno) [inline app/views/self_service/firms/questionnaire/_business_split.html.erb]: You timed that one perfectly!

**Comment by benbarnett** [comment]
:+1: too late!

File: app/views/self_service/firms/questionnaire/_business_split.html.erb (hunk 1 above)
Comment (neoeno) [inline app/views/self_service/firms/questionnaire/_business_split.html.erb]: I'll sort it out anyway

Comment (benbarnett) [inline app/views/self_service/firms/questionnaire/_business_split.html.erb]: Haha, I'm going to buy a lottery ticket tonight

**Pull request merged**